dapp-core/src/main/kotlin/jp/co/soramitsu/dapp/cache/DefaultCacheManager.kt

```diff
@@ -0,0 +1,36 @@
+package jp.co.soramitsu.dapp.cache
+
+import jp.co.soramitsu.dapp.helper.CacheManager
+import org.springframework.stereotype.Component
+
+@Component
+class DefaultCacheManager : CacheManager {
```

Comment:
Is this class used in multiple threads? If so, all functions must be marked with `@Synchronized`. And why do we even need this hand-made cache manager? How about [ehcache](https://www.ehcache.org/)?

Reply:
The initial idea was to develop both in-memory and storage-based caches, but I think it will be done later, now we need to get some initial build working somehow. Thank you for the suggestion.